api/test/e2e/info_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package e2e
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestInfo(t *testing.T) {
+	tests := []HttpTestCase{
+		{
+			Desc:         "get info",
+			Object:       ManagerApiExpect(t),
+			Method:       http.MethodGet,
+			Path:         "/info",
+			Headers:      map[string]string{"Authorization": token},
+			ExpectStatus: http.StatusOK,
+			ExpectBody:   []string{"commit_hash", "\"version\":\"master\""},
```

Comment:
where is `commit_hash`'s value?

Reply:
Because the commit_hash is always changing. So we only guarantee the key exists.

Reply:
ok